mono/mini/mini.h

```diff
@@ -315,7 +315,7 @@ enum {
 #define MONO_IS_ZERO(ins) (((ins)->opcode == OP_VZERO) || ((ins)->opcode == OP_XZERO))
 
 #ifdef TARGET_ARM64
-#define MONO_CLASS_IS_SIMD(cfg, klass) (((cfg)->opt & MONO_OPT_SIMD) && COMPILE_LLVM (cfg) && m_class_is_simd_type (klass))
+#define MONO_CLASS_IS_SIMD(cfg, klass) (0)
 #else
```

Comment:
Why is this needed ?

Reply:
I think we should get rid of MONO_ARCH_SIMD_SUPPORTED, its not very clear what it means any more.

Reply:
We should keep `MONO_CLASS_IS_SIMD` as is.

Getting rid of `MONO_ARCH_SIMD_SUPPORTED` would be nice. It doesn't save that much code and it would be nice to unconditionally enable it everywhere and guard few places with runtime check (eg. `mono_arch_is_simd_enabled (cfg)`).

Reply:
Agree.

Reply:
If you don't mind I'll keep it `(0)` as a TODO - won't enable Vector<T> as part of this PR, but at least it prepares stuff and enables ArmBase